docs/developer-guide/printing-module.md

```diff
@@ -0,0 +1,114 @@
+# Printing Module
+
+The **printing module** of MapStore is a back-end service **not included  by default in the release package** that allows to create a printable PDF from the current map.
+
+!!! note
+    The **printing module** is required by the **Print plugin** of MapStore, so if you want to have the Print plugin working in you application, you have to include also the printing module in your MapStore installation.
+
+## Including the printing module in MapStore
+
+Because MapStore doesn't include the printing module by default, to use it you need to build rom the source a `MapStore.war` that includes it or add the missing files to an existing `MapStore.war`
+
+### Building from the Source
+
+If you want to include the printing module in your MapStore, building the source code you have to add the profile `printing` (profiles are the 2nd argument of the `build.sh` script, after the version):
+
+```bash
+./build.sh [version_identifier] printing
+```
+
+Also MapStore projects allow to use the `printing` profile to include this module.
```

Comment:
```suggestion
MapStore projects allow to use the `printing` profile to include this module.
```

Reply:
I wanted to express that MapStore (standard) and MapStore (projects) build scripts can both use the `printing` profile to include the necessary things. What do you thing about this @MV88 
```suggestion
MapStore projects also allow to use the `printing` profile to include this module. So you can use the same `printing` profile to build your custom MapStore project including the printing module.
```